openfe/setup/ligand_network_planning.py

```diff
@@ -17,31 +17,48 @@
 
 from lomap import generate_lomap_network, LomapAtomMapper
 from lomap.dbmol import _find_common_core
+from konnektor.network_planners import (
+    StarNetworkGenerator,
+    MaximalNetworkGenerator,
+    RedundantMinimalSpanningTreeNetworkGenerator,
+    MinimalSpanningTreeNetworkGenerator,
+    ExplicitNetworkGenerator,
+)
+
+# Konnektor handles:
+from konnektor import network_analysis, network_planners, network_tools
 
 
 def _hasten_lomap(mapper, ligands):
-    """take a mapper and some ligands, put a common core arg into the mapper """
+    """take a mapper and some ligands, put a common core arg into the mapper"""
     if mapper.seed:
         return mapper
 
     try:
-        core = _find_common_core([m.to_rdkit() for m in ligands],
-                                 element_change=mapper.element_change)
+        core = _find_common_core(
+            [m.to_rdkit() for m in ligands],
+            element_change=mapper.element_change,
+        )
     except RuntimeError:  # in case MCS throws a hissy fit
         core = ""
 
     return LomapAtomMapper(
-        time=mapper.time, threed=mapper.threed, max3d=mapper.max3d,
-        element_change=mapper.element_change, seed=core,
-        shift=mapper.shift
+        time=mapper.time,
+        threed=mapper.threed,
+        max3d=mapper.max3d,
+        element_change=mapper.element_change,
+        seed=core,
+        shift=mapper.shift,
     )
 
 
 def generate_radial_network(
     ligands: Iterable[SmallMoleculeComponent],
-    central_ligand: Union[SmallMoleculeComponent, str, int],
     mappers: Union[AtomMapper, Iterable[AtomMapper]],
+    central_ligand: Union[SmallMoleculeComponent, str, int] = None,
```

Comment:
This is the wrong type hint and changes the default behaviour. The assumed default behaviour is that you are required to pass a value for central_ligand - I would not change this for 1.x.

One option is to add `None` to the Union, but not set a default value.

P.S. docstring needs updating.